include/mbedtls/mbedtls_config.h

```diff
@@ -2160,6 +2160,14 @@
  */
 #define MBEDTLS_DEBUG_C
 
+/**
+ * \def MBEDTLS_INST
```

Comment:
I think should be called `MBEDTLS_TRACE` (see my comment on `inst.h`).

Reply:
Will do.

Reply:
Renamed both.